### PR TITLE
Player action screen

### PR DESCRIPTION
**Pull request by Ben-Coco**
non-functional player action screen

### CODE REVIEW

File: src/Display/Action.py
Comment (JDrakeFord) [inline src/Display/Action.py:66]: just leaving a comment to block the PR - it isn't quite ready to be merged - Joe and I will work on this soon

**Review by JDrakeFord — CHANGES_REQUESTED** [review]
Needs some changes

we need to make sure this meets all of the requirements for the play action screen before we merge into master

File: src/Display/Action.py
Comment (Ben-Coco) [reply src/Display/Action.py:66]: got it

**Pull request closed without merging**
